store/sqlstore/supplier.go

```diff
@@ -327,6 +330,10 @@ func (ss *SqlSupplier) GetMaster() *gorp.DbMap {
 }
 
 func (ss *SqlSupplier) GetSearchReplica() *gorp.DbMap {
+	if ss.license == nil {
```

Comment:
We need to acquire the mutex on read too, right? Otherwise the mutex isn't really guarding against dirty reads.